frontend/storybook/src/components/entry/imagelabel/admin/images/image-label-right.ts

```diff
@@ -0,0 +1,31 @@
+import {ImageStyle} from "~/components/entry/imagelabel/admin/images/lists/image-style";
```

Comment:
The file location here should match the storybook - `entry/admin/images/meta/...`